sdks/go/test/run_validatesrunner_tests.sh

```diff
@@ -255,6 +255,7 @@ elif [[ "$RUNNER" == "flink" || "$RUNNER" == "spark" || "$RUNNER" == "samza" ||
   if [[ -z "$ENDPOINT" ]]; then
     JOB_PORT=$(python3 -c "$SOCKET_SCRIPT")
     ENDPOINT="localhost:$JOB_PORT"
+    IO_EXPANSION_ADDR="localhost:9000"
```

Comment:
I suspect the issue is you're hard coding a port. Use the same format as JOB_PORT to get a new one, and then use that.